production/helm/meta-monitoring/values.yaml

```
@@ -0,0 +1,165 @@
+cluster:
+  name: loki-meta-monitoring-cluster
+
+# Destinations for telemetry data (metrics, logs)
+# The credentials are stored in the secrets metrics and logs
+# Further authentication methods are supported, see the documentation (https://github.com/grafana/k8s-monitoring-helm/tree/main/charts/k8s-monitoring/docs/examples/auth)
+destinations:
+  - name: prometheus
+    type: prometheus
+    url: https://<PROMETHEUS-ENDPOINT>/api/prom/push
+    auth:
+      type: basic
+      usernameKey: username
+      passwordKey: password
+    secret:
+        create: false
+        name: metrics
+        namespace: meta
+
+  - name: loki
+    type: loki
+    url: https://<LOKI-ENDPOINT>/loki/api/v1/push
+    auth:
+      type: basic
+      usernameKey: username
+      passwordKey: password
+    secret:
+        create: false
+        name: logs
+        namespace: meta
+
+# Components to be monitored by the meta-monitoring Helm chart. 
+# Two integrations are being used:
+# - alloy: https://github.com/grafana/k8s-monitoring-helm/blob/main/charts/k8s-monitoring/charts/feature-integrations/docs/integrations/alloy.md 
+# - loki: https://github.com/grafana/k8s-monitoring-helm/blob/main/charts/k8s-monitoring/charts/feature-integrations/docs/integrations/loki.md
+integrations:
+  collector: alloy-singleton
+  alloy:
+    instances:
+      # monitor the collectors gathering and sending metrics/logs to the local cluster
+      - name: alloy
+        labelSelectors:
+          app.kubernetes.io/name: [alloy-singleton]
+        namespaces:
+          - meta
+
+  loki:
+    instances:
+      - name: loki
+        namespaces:
+          - loki
+        labelSelectors:
+          app.kubernetes.io/name: loki
+        logs:
+          tuning:
+            # extract logfmt fields and set them as structured metadata
+            structuredMetadata:
+              caller:
+              tenant:
+              org_id:
+              user:
+# (Optional) Kubernetes events are captured as logs and are annotated with additional metadata to make them easier to search and filter.
+clusterEvents:
+  enabled: true
+  collector: alloy-singleton
+  namespaces:
+    - meta
+    - loki
+# A collection of metric collectors that gather metrics from various sources in the cluster.
+# (Required) cadvisor - Used to collect Loki pod metrics. Cadvisor is automatically deployed.
+# kubelet - Kubernetes information on each node
+# kubeletResource - Scrape resource metrics from the Kubelet.
+# kube-state-metrics - A simple service that listens to the Kubernetes API server and generates metrics about the state of the objects.
+clusterMetrics:
+  enabled: true
+  collector: alloy-singleton
+  kubelet:
+    enabled: true
+  kubeletResource:
+    enabled: true
+  cadvisor:
+    enabled: true
+    extraMetricProcessingRules: |-
+      rule {
+        action = "keep"
+        source_labels = ["namespace"]
+        regex = "meta|loki"
+      }
```

We just released something to make this built-in!
https://github.com/grafana/k8s-monitoring-helm/blob/main/charts/k8s-monitoring/charts/feature-cluster-metrics/values.yaml#L260-L262
```suggestion
    metricsTuning:
      includeNamespaces:
        - loki
        - meta
```